docs/source/getting_started/command_line_tool.rst

```diff
@@ -39,6 +39,43 @@ If lightly was installed correctly, you should see something like this:
 
     lightly version 1.1.4
 
+Crop Images using Labels or Predictions
+---------------------------------------------------
+For some tasks, self-supervised learning on an image level has disadvantages. For 
+example, when training an object detection model we care about local features
+describing the objects rather than features describing the full image.
+
+One simple trick to overcome this limitation, is to use labels or to use a pre-trained model
+to get bounding boxes around the objects and then cropping the objects out of the
+image.
+
+We can do this using the **lightly-crop** CLI command.  
+
+.. code-block:: bash
+
+    # Crop images and set the crop to be 20% around the bounding box
+    lightly-crop input_dir=images label_dir=labels output_dir=cropped_images crop_padding=0.2
+
+    # Crop images and use the class names in the filename
+    lightly-crop input_dir=images label_dir=labels output_dir=cropped_images \
+                 label_names_file=data.yaml
+
+The labels should be in the yolo format. For each image you should have a
+corresponding .txt file. Each row in the .txt file has the following format:
+
+* class x_center y_center width height
+
+.. code-block:: text
+
+    0 0.23 0.14 0.05 0.04
+    1 0.43 0.13 0.12 0.08
+
+An example for the labe names .yaml file:
+
+.. code-block:: yaml
+
+    names: [cat, dog]
+
```

Comment:
You could add one line, that the output-dir of `lightly-crop` can be used as `input_dir` for the next command, e.g.
`lightly-train input_dir=cropped_images ...`